automation/run-tests.sh

```diff
@@ -85,8 +85,7 @@ function run_tests {
     if [ $TEST_TYPE == $TEST_TYPE_ALL ] || \
        [ $TEST_TYPE == $TEST_TYPE_UNIT_PY36 ];then
         if [[ $CONTAINER_IMAGE == *"centos"* ]]; then
-            echo "Running unit test in $CONTAINER_IMAGE container is not " \
-                 "support yet"
+            container_exec 'tox --sitepackages -e py36'
```

Comment:
Having ‘sitepackages’ set externally requires the caller of tox to know about it, therefore I prefer it inside the tox file.
As we usually use the script, I can live with this too.

Reply:
The setting only works if the Python version is the same as the base system. So on Fedora 31, it would not work with sitepackages, since it uses Python 3.7. There one would need to install the build deps for gobject/dbus to be able to run the unit tests.